File: languages/erlang/fibonacci.erl
-module(fibonacci).
-export([start/0]).

start() ->
  [V|_] = init:get_plain_arguments(),
  {N, _} = string:to_integer(V),
  io:format("~p\n", [fib(N - 1)]).

fib(N) when N < 2 -> N;
fib(N) -> fib(N-1) + fib(N-2).

File: languages/java/Fibonacci.java
class Fibonacci {
	
	public static void main(String[] args){
		System.out.println(fib(Integer.parseInt(args[0]) - 1));
	}
	
	public static int fib(int n){
		if (n < 2)
			return n;
		return fib(n - 1) + fib(n - 2);
	}
	
}